### PR TITLE
feat: remove notional total

### PR DESCRIPTION
**Pull request by gpuente**
## Description:

- remove notional total from RWA Assets Table

### CODE REVIEW

File: src/rwa/components/table/assets/assets-table.tsx [hunk 1]
```diff
@@ -219,12 +214,10 @@ export function AssetsTable(props: AssetsTableProps) {
                         </RWATableCell>
                     )}
                     {column.key !== 'name' &&
-                        column.key !== 'notional' &&
                         column.key !== 'purchaseProceeds' &&
                         column.key !== 'salesProceeds' &&
                         column.key !== 'totalDiscount' &&
-                        column.key !== 'realizedSurplus' &&
-                        column.key !== 'currentValue' && (
```
Comment (acaldas) [inline src/rwa/components/table/assets/assets-table.tsx:227]: Was the currentValue deleted on purpose?

**Review by acaldas — APPROVED** [review]

File: src/rwa/components/table/assets/assets-table.tsx (hunk 1 above)
Comment (gpuente) [reply src/rwa/components/table/assets/assets-table.tsx:227]: yes, I think it wasn't removed previously. This was causing a missing cell.

Comment (acaldas) [reply src/rwa/components/table/assets/assets-table.tsx:227]: But the currentValue is still shown with this change?

**Review by acaldas — APPROVED** [review]

File: src/rwa/components/table/assets/assets-table.tsx (hunk 1 above)
Comment (gpuente) [reply src/rwa/components/table/assets/assets-table.tsx:227]: yes, this is for the totals row. Previously we were rendering null for `currentValue` total cell, causing an offset (the rest of total values where displayed in the incorrect column). now `currentValue` is handled by this last change and renders an empty cell (just like all the other cells that does not require a total).

**Pull request merged**